groups.go

```diff
@@ -44,6 +45,12 @@ func (srv *groupsAPI) CreateGroup(ctx context.Context, in *accountsv1.CreateGrou
 		return nil, status.Error(codes.Internal, err.Error())
 	}
 
+	conversation := models.ConversationInfo{Title: "Default conversation", GroupID: group.ID}
```

Comment:
I think we should put a name other than "Default Conversation". Maybe "General"?. Also you should call the `CreateConversation` endpoint instead of making the repository call.
> This endpoint is not used cause everything is done is in CreateGroup endpoint, don't know if i can call an endpoint in another endpoint

Yes you can do that! Let me know if you need me to show you how.